From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Harmer <mark.harmer@discordapp.com>
Date: Sun, 26 Feb 2023 12:03:55 -0800
Subject: fix: Revert
 https://chromium-review.googlesource.com/c/chromium/src/+/3118293

This was causing the early-exit to be skipped within SynchronizeVisualProperties
when there was an existing pending ack. This in turn would cause these visual
property updates to be queued up. On Mac devices this would cause visible
delays in rendering when the window was resized often.


diff --git a/content/browser/renderer_host/render_widget_host_impl.cc b/content/browser/renderer_host/render_widget_host_impl.cc
index 6ed422b4bfc50..aedb26a4ff148 100644
--- a/content/browser/renderer_host/render_widget_host_impl.cc
+++ b/content/browser/renderer_host/render_widget_host_impl.cc
@@ -2169,9 +2169,8 @@ RenderWidgetHostImpl::GetWidgetInputHandler() {
 void RenderWidgetHostImpl::NotifyScreenInfoChanged() {
   // The resize message (which may not happen immediately) will carry with it
   // the screen info as well as the new size (if the screen has changed scale
-  // factor). Force sending the new visual properties even if there is one in
-  // flight to ensure proper IPC ordering for features like the Fullscreen API.
-  SynchronizeVisualPropertiesIgnoringPendingAck();
+  // factor).
+  SynchronizeVisualProperties();
 
   // The device scale factor will be same for all the views contained by the
   // primary main frame, so just set it once.

